utils/dataimport.py

```diff
@@ -14,8 +14,28 @@ class ParseError(Exception):
     """Raised when the data has invalid value"""
     pass
 
+
+"""
+This class contains static members to import data to Mulysa. Data can be:
+
+ - Member data. Current implementation imports CSV file used by old
+   Tampere hacklab registry. It is only for reference - write your
+   own importer!
+
+ - Bank events. This is in form of bank transactions in and out of an
+   account. Only one account is supported currently.
+   Current implementation uses Nordea TITO format which seems to be
+   some kind of standard.
+   Use it if you can or write your own importer.
+
+After data is imported, businesslogic to update all user data is
+invoked. All functions return an object showing counts of imported,
+already existing and failed items and array of failed rows. These
+are displayed in the import UI.
+"""
 class DataImport:
 
+    # Import cvs member list. Only for reference.
```

Comment:
while you are here might want to check this also: https://github.com/TampereHacklab/mulysa/pull/130/files#diff-52dcbb19524880e4f0a4f6ccbcaade9bR42

#118 has untested better code for the split